stable/prometheus-adapter/README.md

```diff
@@ -51,6 +51,7 @@ The following table lists the configurable parameters of the Prometheus Adapter
 | `rules.default`                 | If `true`, enable a set of default rules in the configmap                       | `true`                                      |
 | `rules.custom`                  | A list of custom configmap rules                                                | `[]`                                        |
 | `rules.existing`                | The name of an existing configMap with rules. Overrides default and custom.     | ``                                          |                                                                                                        
```

Comment:
Should maybe mention `external` now as well?

Reply:
@Multiply It does mentioned in line 54

Reply:
I know, but this line mentions that `default` and `custom` is overriden, not `external`.

Reply:
Updated, thank you @Multiply